modules/console/pages/config/connect-to-redpanda.adoc

```diff
@@ -1,93 +1,220 @@
-      password: "my-password"
-      realm: "MY.REALM"
+      username: "example-user"
+      password: "example-password"
+      realm: "EXAMPLE.REALM"
       enableFast: true
 ----
 
+NOTE: Because Redpanda Console supports only OIDC and basic authentication for its own login authentication, Kerberos (GSSAPI) configurations cannot be used with user impersonation enabled. Ensure `impersonateUser` is set to `false` when using Kerberos.
+
+[cols="1a,4a"]
+|===
+| Option | Description
+
+| `brokers`
+| A list of broker endpoints in the format `<hostname>:<port>`. For high availability, include all broker URLs.
+
+| `clientId` (optional)
+| An identifier used by Redpanda Console to identify itself to the cluster.
+
+| `rackId` (optional)
+| Specifies the rack for multi-zone clusters to optimize local message consumption.
+
+| `sasl` (optional)
+| Contains settings for SASL-based authentication. Configure this block if your cluster requires authentication through one of the following mechanisms:
+
+- **Basic authentication:** Provide `username` and `password`.
+- **OAuth 2.0:** Provide `token` to use a static token or provide `clientId`, `clientSecret`, `tokenEndpoint`, and `scope` to acquire new tokens at runtime.
+- **Kerberos (GSSAPI):** Provide parameters such as `authType`, `keyTabPath`, `kerberosConfigPath`, `serviceName`, `username`, `password`, `realm`, and `enableFast`.
+
+Supported mechanisms:
+
+- PLAIN
+- SCRAM-SHA-256 or SCRAM-SHA-512
+- GSSAPI
+- OAUTHBEARER
+
+NOTE: Because Redpanda Console supports only OIDC and basic authentication for its own login authentication, Kerberos (GSSAPI) configurations must be used with `impersonateUser` set to `false`.
+
+For more detailed instructions on configuring authentication, see xref:console:config/security/authentication.adoc[].
+
+| `startup` (optional)
+| Controls connection behavior at startup:
+
+- `establishConnectionEagerly`: Tests the connection immediately.
+- `maxRetries`, `retryInterval`, `maxRetryInterval`, `backoffMultiplier`: Define the retry logic for establishing a connection.
+
+| `tls` (optional)
+| Contains settings to secure the connection using TLS. Specify paths for the CA certificate, client certificate, and client key. Optionally configure `insecureSkipTlsVerify` for testing purposes.
+|===
+
+== Configure access to the Schema Registry
+
+To enable schema management features, you must configure Redpanda Console to connect to the Schema Registry API. This includes specifying the service endpoints and, if needed, setting up authentication.
+
+Example Schema Registry configuration:
+
+[source,yaml]
+----
+schemaRegistry:
+  enabled: true
+  urls:
+    - "broker1.example.com:8081"
+    - "broker2.example.com:8081"
+
+  # Optional authentication settings
+  authentication:
+    impersonateUser: true
+    #basic:
+      #username: "example-user"
+      #password: "example-password"
+    #bearerToken: "example-bearer-token"
+----
+
+[cols="1a,4a"]
+|===
+| Option | Description
```

Comment:
```suggestion
| Schema Registry Configuration Option | Description
```

Reply:
When scrolling around with these tables, it was easy for me to lose context, and I kept having to check the right sidenav. Given that these tables have no headers that identify them clearly, having the specific config name in the column headings seems like a simple way to provide immediate context without making the reader have to look anywhere else.